Drop-In/src/main/res/layout/bt_fragment_select_payment_method_parent.xml

```diff
@@ -10,6 +10,7 @@
         android:id="@+id/background"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:alpha="0"
```

Comment:
Not sure if this alpha is the cause or something else but the AddCardFragment is now transparent

Reply:
Nice yeah that's my b. There were two things from that rebase that didn't make it. We have to explicitly set backgrounds on the individual fragments now since the activity is transparent.